### PR TITLE
Update Washington_Times.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Washington_Times.xml [hunk 1]
```diff
@@ -1,29 +1,68 @@
...
-	Nonfunctional domains:
+	Non-functional hosts
+		Couldn't connect to server:
+		- advstage.washingtontimes.com
+		- classified.washingtontimes.com
+		- directory.washingtontimes.com
+		- email.washingtontimes.com
+		- webmail.washingtontimes.com
 
-		- media.washtimes.com *
```
Comment (J0WI) [inline src/chrome/content/rules/Washington_Times.xml]: Could you add `*.washtimes.com` somewhere?

File: src/chrome/content/rules/Washington_Times.xml [hunk 2]
```diff
@@ -1,29 +1,68 @@
...
 
+		Self-signed certificate chain error:
+		- paywaytest.washingtontimes.com
 
-	<securecookie host="^(?:iservices|subscription)\.washingtontimes\.com$" name=".+" />
-
-
-	<rule from="^http://(iservices|subscription)\.washingtontimes\.com/"
-		to="https://$1.washingtontimes.com/" />
+		4xx client error:
```
Comment (J0WI) [inline src/chrome/content/rules/Washington_Times.xml]: A custom test url should work here.

**Comment by cschanaj** [comment]
@J0WI I have added the `target` and a new ruleset for `*washtimes.com`. Thanks!!

File: src/chrome/content/rules/Washtimes.com.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/Washtimes.com.xml]: Please add referees to each other.

**Pull request merged**